.github/workflows/build.yaml

```diff
@@ -52,11 +52,17 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Get current day
+        uses: josStorer/get-current-time@v2.0.1
+        id: current-day
+        with:
+          format: YYYY-MM-DD
```

Comment:
do we just want the day, or the time?

Reply:
We just want the day–we want to bust the cache automatically every day

Reply:
An alternate approach we were discussing this morning would be to include the base branch’s sha (head for the branch) in the key so we don’t end up with stale docker layers in long lived PRs and every merge to main busts the cache for every long-lived PR.

Reply:
What do you mean by "bust"? Do you want to rebuild without cache daily, or write to the cache daily?

Reply:
As an alternate alternate approach, we could delete all of the repo's cache on a nightly basis. This is now possible via the API

Reply:
We want to ignore the cache if the cache was created before today. A nightly and/or manually triggered workflow to actually delete the cache would be a better option. Stale cache currently blocking another PR so I'd lean towards getting both a quick simple fix in now and a plan to get a well thought out fix going in parallel.

Reply:
^ @VB-at-Bis

Reply:
hmm.. https://docs.github.com/en/rest/actions/cache#delete-github-actions-caches-for-a-repository-using-a-cache-key looks promising.. let me see if I can find key etc

Reply:
So do we want to continue with this PR or close it in favour of the deletes? I'm leaning to close

Reply:
I added comment to the ticket with steps how to use GitHub API to clear the cached image.